appcues/src/main/java/com/appcues/analytics/DefaultActivityBuilder.kt

```diff
@@ -0,0 +1,148 @@
+package com.appcues.analytics
+
+import android.os.Build.VERSION
+import com.appcues.AnalyticType
+import com.appcues.AnalyticType.EVENT
+import com.appcues.AnalyticType.GROUP
+import com.appcues.AnalyticType.IDENTIFY
+import com.appcues.AnalyticType.SCREEN
+import com.appcues.AppcuesConfig
+import com.appcues.BuildConfig
+import com.appcues.R
+import com.appcues.analytics.AnalyticsIntent.Anonymous
+import com.appcues.analytics.AnalyticsIntent.Event
+import com.appcues.analytics.AnalyticsIntent.Identify
+import com.appcues.analytics.AnalyticsIntent.Screen
+import com.appcues.analytics.AnalyticsIntent.UpdateGroup
+import com.appcues.analytics.AnalyticsIntent.UpdateProfile
+import com.appcues.util.ContextResources
+import java.util.Date
+
+internal class DefaultActivityBuilder(
```

Comment:
class is responsible for mapping AnalyticsIntent to AnalyticsActivity, this means simplifying from Anonymous, Identify and UpdateProfile to the AnalyticType.IDENTIFY for instance. + adding the necessary properties, attributes and context values